cli/client.go

```diff
@@ -1368,6 +1368,21 @@ func RobotsPartTunnelAction(c *cli.Context, args robotsPartTunnelArgs) error {
 }
 
 func tunnelTraffic(ctx *cli.Context, robotClient *client.RobotClient, local, dest int) error {
+	// don't block tunnel attempt if ListTunnels fails in any way - it may be unimplemented.
+	// TODO: early return if ListTunnels fails.
```

Comment:
maybe do in a month

Reply:
You don't want to this in actual `robotClient.Tunnel`? That way both the CLI and anyone trying to programmatically use `robotClient.Tunnel` both encounter early errors?

Reply:
issue is `robotClient.Tunnel` isn't called until the first connection is made. if `robotClient.Tunnel` is called, it'd error out immediately even without this change

Reply:
Ah gotcha thanks for the explanation 🙏🏻 .